teamprojekt_competition_server/server/interfaces.py

```diff
@@ -53,18 +53,19 @@ def notify_end(self, result, stats):
 class IGame(abc.ABC):
     """game interface"""
 
-    def __init__(self, players: list[IPlayer]) -> None:
+    def __init__(self, players: list[IPlayer], game_id: int = 123) -> None:

```

Comment:
I think it would be better, to not set the game_id to a standard dummy value, because it is now possible to create a game without noticing, the `game_id` has still a dummy value. It would be better to write the dummy value where the game is created.

We haven't decided yet, if the `game_manager` will write into the database or the game will write itself in the database. In the second case it would not make sense to initialize the game with the `game_id`. But this is fine by now.

Reply:
ok. I agree so removed it